packages/playwright/src/index.ts

```diff
@@ -322,6 +322,8 @@ const playwrightFixtures: Fixtures<TestFixtures, WorkerFixtures> = ({
     clientInstrumentation.removeListener(csiListener);
     await artifactsRecorder.didFinishTest();
 
+    if (process.env.PLAYWRIGHT_COPY_PROMPT && (testInfo as TestInfoImpl)._isFailure())
+      await testInfo.attach('testSource', { path: testInfo.file, contentType: 'application/javascript' });
```

Comment:
nit: We have the test location (line/column) here, but we don't store it and it feels wrong.